db/fixtures/ae_datastore/Deployment/System/StateMachine/Deployment.class/__methods__/cleanup.rb

```diff
@@ -0,0 +1,11 @@
+def cleanup
+  $evm.log(:info, "********************** clean_up ***************************")
+
+  $evm.root['state'] = "clean_up"
```

Comment:
@alongoldboim 
There is already a state field stored in $evm.root called $evm.root['ae_state'] which is a readonly field for the methods. Can we use that if you are using it only for reading the current state